### PR TITLE
STN-199 polish integration

### PR DESCRIPTION
**Pull request by heatherdesigns**
# [STN-199](https://sparkbox.atlassian.net/browse/STN-199) READY FOR REVIEW

## Summary
This PR includes a portion of design refinements documented in [STN-199](https://sparkbox.atlassian.net/browse/STN-199).

Designer: @meranicosme 
Developer: @heatherdesigns 

## Need Review By (Date)
EOD 3/16/2020

## Urgency
high

## Steps to Test
1. In the CLI, run `npm run test` to verify all tests pass.
2. Go to http://economics.suhumsci.loc. Verify the following design refinements have been implemented:
 - [x] Darken base secondary mint color to: #148762. Passes color contrast
 - [x] Make sure people component has a border bottom: Driftwood color: #B6B1A9
 - [x] Main Navigation Dropdown links should be driftwood and then on hover change to black
 - [x] Main Navigation: Links should be font-weight: semibold
 - [x] Hero 1: Check the vertical size for mobile. Looks a bit too long but double check (height was adjusted0
 - [x] Speed up navigation arrow animation on desktop
 - [x] Add drop shadow to desktop nav dropdown and change to driftwood and 1px width
 - [x] Desktop navigation border bottom should be driftwood and 1px

## PR Checklist
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
- [Sparkbox PR Checklist](../docs/SparkboxPRChecklist.md)

### CODE REVIEW

File: docroot/themes/humsci/humsci_basic/src/scss/components/_main-nav.scss [hunk 1]
```diff
@@ -148,6 +152,10 @@
     text-decoration: none;
     line-height: 120.3%;
 
+    // .hb-main-nav__menu-lv2 & {
+    //   color: hotpink; // WIP
+    // }
```
Comment (heatherdesigns) [inline docroot/themes/humsci/humsci_basic/src/scss/components/_main-nav.scss]: Ooops!!! Note to self: remove this!!!

Comment (silverli) [reply docroot/themes/humsci/humsci_basic/src/scss/components/_main-nav.scss]: :)

File: docroot/themes/humsci/humsci_basic/src/scss/components/_pattern.hero.scss [hunk 2]
```diff
@@ -110,7 +110,11 @@
         height: 100%;
         width: 100%;
         object-fit: cover;
-        min-height: 55vh;
+        min-height: 35vh; // was 55vh
+
+        @include grid-media-min('md') {
+          min-height: 55vh; // WIP
```
Comment (heatherdesigns) [inline docroot/themes/humsci/humsci_basic/src/scss/components/_pattern.hero.scss]: Remove `// WIP`

File: docroot/themes/humsci/humsci_basic/src/scss/components/_pattern.hero.scss [hunk 3]
```diff
@@ -110,7 +110,11 @@
         height: 100%;
         width: 100%;
         object-fit: cover;
-        min-height: 55vh;
+        min-height: 35vh; // was 55vh
```
Comment (heatherdesigns) [inline docroot/themes/humsci/humsci_basic/src/scss/components/_pattern.hero.scss]: Remove comment.

File: docroot/themes/humsci/humsci_basic/src/scss/components/_main-nav.scss [hunk 4]
```diff
@@ -171,6 +179,7 @@
 
     &:hover:not(.is-active) {
```
Comment (silverli) [inline docroot/themes/humsci/humsci_basic/src/scss/components/_main-nav.scss:172]: hmm we might need a focus style here too?

File: docroot/themes/humsci/humsci_basic/src/scss/components/_structured-card.scss [hunk 5]
```diff
@@ -1,5 +1,7 @@
 .hb-card--structured {
   width: 100%;
+  border-bottom: 1px solid $su-color-driftwood;
```
Comment (silverli) [inline docroot/themes/humsci/humsci_basic/src/scss/components/_structured-card.scss:3]: 👍

File: docroot/themes/humsci/humsci_basic/src/scss/elements/_base.scss [hunk 6]
```diff
@@ -114,6 +114,10 @@ button:not([class]) {
 // config/default/extlink.settings.yml
 // and replace with different icon
 .fa-ext {
+  // scss-lint:disable ImportantRule
+  margin-right: 0 !important; // removes link underline from icon
```
Comment (silverli) [inline docroot/themes/humsci/humsci_basic/src/scss/elements/_base.scss:118]: good comment

File: docroot/themes/humsci/humsci_basic/src/scss/settings/_variables.general.scss [hunk 7]
```diff
@@ -30,7 +30,7 @@ $hb-sidebar-width: 22.5%;
 $hb-border-radius: 4px;
 
 // Menu Variables
-$hb-menu-light-border: 2px solid $hb-gray--90;
+$hb-menu-light-border: 1px solid $su-color-driftwood;
```
Comment (silverli) [inline docroot/themes/humsci/humsci_basic/src/scss/settings/_variables.general.scss:33]: 👍

File: docroot/themes/humsci/humsci_basic/src/scss/settings/_variables.theme.colors.scss [hunk 8]
```diff
@@ -21,12 +21,14 @@ $hb-gray--10: hb-gray(10);
 $hb-color--black: hb-gray(0);
 $hb-color--white: hb-gray(100);
 
+$hb-color--mint: #148762; // darken $sb-color-mint to pass color contrast
```
Comment (silverli) [inline docroot/themes/humsci/humsci_basic/src/scss/settings/_variables.theme.colors.scss]: *darken $su-color-mint

**Review by silverli — CHANGES_REQUESTED** [review]
this is looking really great.  I tested everything in safari, ie11, firefox, and chrome. 

We do need to fix some linting errors though, and I made a couple comments in the code.

Also... question that we should maybe or maybe not address here, but should the active (green) list item also get text color darkened to black?:
Screen Shot 2020-03-16 at 4 09 36 PM

File: docroot/themes/humsci/humsci_basic/src/scss/components/_main-nav.scss (hunk 4 above)
Comment (heatherdesigns) [reply docroot/themes/humsci/humsci_basic/src/scss/components/_main-nav.scss:172]: See line 224. Adding the focus state here darkens the top level nav items. I `focused` on the dropdown nav items. :smile:

**Review by silverli — APPROVED** [review]
awesome work!!

**Pull request merged**